arch/arm64/boot/dts/aspeed/aspeed-bmc-amd-congo.dts

```diff
@@ -539,3 +539,9 @@
 &phy3a {
 	status = "okay";
 };
+
+&lpc0_kcs2 {
```

Comment:
Can you cleanup this

Reply:
facing issues with rebasing moved to new PR.

https://github.com/AMDESE/linux-aspeed/pull/28